.travis.yml

```diff
@@ -46,26 +53,33 @@ before_install:
     conda update conda
 
 install:
-  - conda env create --file environment.yml --name BiG-CZ
-  - source activate BiG-CZ
+  # We need to envs b/c the BiG-CZ software stack is not compatible with Pyhton 3 yet.
+  - conda env create --file webpage/environment.yml --name PAGE
+  - conda env create --file environment.yml --name BiG-CZ ;
 
   # Debug.
   - conda info --all
   - conda list
 
 script:
   - if [ $TEST_TARGET == "coding_standards" ]; then
+      source activate BiG-CZ ;
       flake8 --max-line-length=100 ;
       python tests/test_coding_standards.py ;
     fi
 
   # Test python notebooks.
   - if [ $TEST_TARGET == "default" ]; then
+      source activate BiG-CZ ;
       python tests/test_notebooks.py ;
     fi
 
   # Publish the notebooks.
   - if [ $TEST_TARGET == "publish" ]; then
+      source activate BiG-CZ ;
```

Comment:
I could activate only once but I decided to be explicit when we are using each env.